data/transport missions.txt

```diff
@@ -1951,3 +1951,247 @@ ship "Science Drone"
 		"heat dissipation" 1
 	explode "tiny explosion" 5
 	explode "small explosion" 2
+
+
+
+mission "Northern Blockade"
+	minor
+	name `Mebsuta Disaster`
+	description `<destination> is under attack by pirates. Assist the Republic by delivering <commodity> to the planet as soon as possible.`
+	source
+		near "Mebsuta" 2 4
+	destination "Featherweight"
+	cargo "relief supplies" 178
+	to offer
+		random < 20
+		"combat rating" > 200
+		"reputation: Republic" >= 0
+	
+	on offer
+		conversation
+			`You are approached by a Navy officer. "Are you Captain <last>?" You nod. "Would you be able to assist us? A sizable gang of pirates has occupied <system> and raided the spaceport. A Navy fleet is on its way to eliminate the pirates, but reports from the planet indicate that the spaceport of <planet> is in dire need of supplies. Any building above three stories was badly damaged and everything in the warehouses was either stolen or destroyed. We need a ship to bring <cargo> to them as soon as possible, but there are no available Navy ships capable of doing the job. Would you deliver the supplies for us, Captain?"`
+			choice
+				`	"I'd be glad to help, Officer."`
+					accept
+				`	"Sorry, but I'm not travelling in that direction."`
```

Comment:
"Travelling" should be "traveling" here. Both spellings are correct, but the rest of the game uses the one 'l' (American) spelling. Will mark other instances as well.

Reply:
Should `travelled` in line 145 of this file be changed to `traveled` then? (That's outside of these missions so it could use its own PR.)

Reply:
No, because two 'l's is correct for the past tense. Thanks again English!

Reply:
You sure? https://www.grammarly.com/blog/traveling/

Reply:
Ugh, looks like that one is choose your own adventure as well. Figure we should go with one 'l' everywhere? I have no objection.